AEPCore/Sources/eventhub/Extension.swift

```diff
@@ -115,6 +115,39 @@ public extension Extension {
         return runtime.getSharedState(extensionName: extensionName, event: event, barrier: true)
     }
 
+    // MARK: - XDM Shared State
```

Comment:
Forgot to add these in the previous XDM shared state PR.

Reply:
Can you split this in a separate PR so we can update dev with this and merge the Identity changes in a feature branch?

Reply:
Created https://github.com/adobe/aepsdk-core-ios/pull/450 and changed the target for this PR to the xdm feature branch. I'll get the XDM feature branch sync'd with dev once the change to dev land.